Sources/Sentry/SentryUIApplication.m

```diff
@@ -56,6 +56,90 @@ - (UIApplication *)sharedApplication
     return result;
 }
 
+- (NSArray<UIViewController *> *)relevantViewControllers
```

Comment:
`m`: I'm a bit worried that there are no tests for this.

Reply:
I believe we need UI Tests for this.